geode-core/src/test/java/org/apache/geode/internal/DataSerializableJUnitTest.java

```diff
@@ -2255,63 +2259,83 @@ public void testSupportedClasses() throws Exception {
     try {
       Object o = new NonDataSerializable(new Random());
       DataSerializer.writeObject(o, getDataOutput());
-      assertTrue(Class_testSupportedClasses2.wasInvoked);
-      assertTrue(Class_testSupportedClasses2.toDataInvoked);
-      assertFalse(Class_testSupportedClasses2.fromDataInvoked);
+      assertThat(Class_testSupportedClasses2.wasInvoked).isTrue();
+      assertThat(Class_testSupportedClasses2.toDataInvoked).isTrue();
+      assertThat(Class_testSupportedClasses2.fromDataInvoked).isFalse();
 
       Object o2 = DataSerializer.readObject(getDataInput());
-      assertTrue(Class_testSupportedClasses2.fromDataInvoked);
-      assertEquals(o, o2);
+      assertThat(Class_testSupportedClasses2.fromDataInvoked).isTrue();
+      assertThat(o).isEqualTo(o2);
     } finally {
       InternalDataSerializer.unregister(id);
       InternalDataSerializer.unregister(id2);
     }
+
+    Class_testSupportedClasses2.wasInvoked = false;
```

Comment:
It would be nice to move this to an "@After" or a "@Before" and get rid of the duplicate code.

Reply:
It is not exactly the same code. Each of the three failing tests is changing a different static class. `testSupportedClasses` is using `Class_testSupportedClasses2`, `testUDDS2` uses `Class_testSupportedClasses3` and `testUDDS4` uses `Class_testSupportedClasses4`.

Reply:
I mean, I could move the code to an `@After` method, but it will not remove any code, I should do the same calls anyway.